tests/acceptance/TestHelpers/CollaborationHelper.php

```diff
@@ -64,4 +64,35 @@ public static function sendPOSTRequestToAppOpen(
 			['Content-Type' => 'application/json']
 		);
 	}
+
+	/**
+	 * @param string $baseUrl
+	 * @param string $xRequestId
+	 * @param string $user
+	 * @param string $password
+	 * @param string $parent_container_id
+	 * @param string $file
+	 * @param array|null $headers
+	 *
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public static function createFileUsingWopiEndpoint(
+		string $baseUrl,
+		string $xRequestId,
+		string $user,
+		string $password,
+		string $parent_container_id,
```

Comment:
Why snake_case rather than camelCase?

Reply:
Just forgot about it. :joy:, because of the same style in the endpoint uri.